lightpath/__main__.py

```diff
@@ -60,8 +63,12 @@ def entrypoint():
     else:
         hutches = None
 
+    # set up simulated lightpath if requested
+    if args.sim:
```

Comment:
`if args.sim and not args.db` (in case I want my own simulation database?)

Reply:
There's nothing special about the sim version of the app, so if you want your own test database you can just pass it in through --db.  If we do implement the "sim" mode with fake devices, this would make sense though.

Reply:
Oh, _right_...